src/wasm-builder.h

```diff
@@ -142,7 +142,7 @@ class Builder {
   }
 
   static std::unique_ptr<Export>
-  makeExport(Name name, Name value, ExternalKind kind) {
```

Comment:
It would be good to have an assertion that the value is always a `Name` for now. In the next PR, that assertion can be updated to check that the type of `value` matches `kind`.